src/psyclone/tests/domain/lfric/lfric_scalar_mdata_test.py

```diff
@@ -308,6 +343,34 @@ def test_arg_descriptor_int_scalar():
     assert scalar_descriptor.vector_size == 0
 
 
+def test_arg_descriptor_logical_scalar():
```

Comment:
Again, I think you could save duplication by parameterising the test that currently does the integer scalar argument.

Reply:
Done. It is a pleasure deleting duplicated code 😄